src/main/java/uk/gov/laa/ccms/caab/bean/NotificationSearchCriteria.java

```diff
@@ -1,6 +1,8 @@
 package uk.gov.laa.ccms.caab.bean;
 
 import lombok.Data;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
import org.springframework.util.StringUtils; 

can we use this one instead?
might be worth removing all the apache ones as well, only see 6 references to it

Reply:
Switched all of them, in one case replaced with plain java (`StringUtils.repeat()` -> `str.repeat()`) as this wasn't provided by the Spring version. Also replaced some deprecated `isEmpty()`.